cli/script/command-executor.ts

```diff
@@ -880,17 +885,23 @@ function promote(command: cli.IPromoteCommand): Promise<void> {
 }
 
 function patch(command: cli.IPatchCommand): Promise<void> {
-    var isMandatory: boolean = getIsMandatoryValue(command.mandatory);
     var packageInfo: PackageInfo = {
         description: command.description,
-        isMandatory: isMandatory,
+        isMandatory: getYargsBooleanOrNull(command.mandatory),
+        isDisabled: getYargsBooleanOrNull(command.disabled),
         rollout: command.rollout
     };
 
-    return sdk.patchRelease(command.appName, command.deploymentName, command.label, packageInfo)
-        .then((): void => {
-            log(`Successfully updated the "${command.label ? command.label : `latest`}" release of "${command.appName}" app's "${command.deploymentName}" deployment.`);
-        });
+    for (var updateProperty in packageInfo) {
```

Comment:
Doesn't this now hit the server multiple times unnecessarily if you specify more than one property to patch?